case/src/test/kotlin/com/ritense/case/deployment/CaseTabDeployerIT.kt

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright 2015-2023 Ritense BV, the Netherlands.
+ *
+ * Licensed under EUPL, Version 1.2 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://joinup.ec.europa.eu/collection/eupl/eupl-text-eupl-12
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" basis,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.ritense.case.deployment
+
+import com.ritense.case.BaseIntegrationTest
+import com.ritense.case.domain.CaseTabType
+import com.ritense.case.repository.CaseTabRepository
+import com.ritense.case.repository.CaseTabSpecificationHelper.Companion.TAB_ORDER
+import com.ritense.case.repository.CaseTabSpecificationHelper.Companion.byCaseDefinitionName
+import com.ritense.valtimo.changelog.repository.ChangesetRepository
+import com.ritense.valtimo.changelog.service.ChangelogDeployer
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.mockito.kotlin.whenever
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.data.domain.Sort
+import org.springframework.transaction.annotation.Transactional
+import java.time.Instant
+
+@Transactional
+class CaseTabDeployerIT : BaseIntegrationTest() {
+    @Autowired
```

Comment:
For integration tests, it makes more sense to autowire these beans in the constructor

Reply:
Changed